miner/worker_builder.go

```diff
@@ -496,6 +496,13 @@ func (w *worker) simulateBundle(
 		}
 	}
 
+	// prune bundle when all txs are dropped
+	if len(bundle.Txs) == 0 {
+		log.Warn("prune bundle", "hash", bundle.Hash().String(), "err", "empty bundle")
+		w.eth.TxPool().PruneBundle(bundle.Hash())
```

Comment:
can you double check this, becasue `bundle.Hash()` will change after you remove one of the tx, so that it can not match previous hash

Reply:
double checked, bundle.Hash() will load the static `hash` field, it would not change after initialization no matter how the bundle txs change.